bin/resty

```diff
@@ -613,6 +615,23 @@ for my $var (sort keys %ENV) {
     $env_list .= "env $var;\n";
 }
 
+my $sa_restart_signals;
+{
+    my $i = 0;
+    my @sigs;
+
+    foreach $name (split(' ', $Config{sig_name})) {
+        $signo{$name} = $i;
+        $i++;
+    }
+
+    for my $sig (qw/ HUP ALRM USR1 USR2 CHLD WINCH /) {
+        push @sigs, $signo{$sig};
+    }
+
+    $sa_restart_signals = join ', ', @sigs;
+}
```

Comment:
Noticing the recent startup time improvements for resty-cli by you @agentzh, I tested this locally to make sure that it added no significant delay.